src/Tooltip.js

```diff
@@ -5,26 +5,28 @@ import PropTypes from 'prop-types';
 import 'react-tippy/dist/tippy.css';
 import { Tooltip as TippyTooltip } from 'react-tippy';
 
-import Paragraph from './Paragraph';
+import theme from './theme';
```

Comment:
Looks like you're getting some linting errors around this on CI. I think changing this to `import { borderRadius } from './theme'` should resolve all the linting errors you're seeing.